### PR TITLE
Translation of submission-forms to _cs

### PR DESCRIPTION
**Pull request by jr-rk**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Analysis
(Write here, if there is needed describe some specific problem. Erase it, when it is not needed.)
## Problems
(Write here, if some unexpected problems occur during solving issues. Erase it, when it is not needed.)

### CODE REVIEW

File: dspace/config/submission-forms_cs.xml [hunk 1]
```diff
@@ -405,65 +390,9 @@
                 </field>
             </row>
         </form>
-
+        <!--/ TRANSLATE THIS -->
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Remove this message

File: dspace/config/submission-forms_cs.xml [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
     <form-definitions>
         <!-- Form used for entering in Bitstream/File metadata after uploading a file -->
+        <!-- TRANSLATE THIS -->
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Remove

File: dspace/config/submission-forms_cs.xml [hunk 3]
```diff
@@ -1575,6 +1504,7 @@
             </row>
         </form>
 
+        <!-- THIS -->
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Remove

File: dspace/config/submission-forms_cs.xml [hunk 4]
```diff
@@ -2089,6 +2043,27 @@
...
+            <row>
+                <field>
+                    <dc-schema>local</dc-schema>
+                    <dc-element>bitstream</dc-element>
+                    <dc-qualifier>redirectToURL</dc-qualifier>
+                    <repeatable>false</repeatable>
+                    <label>Upload file from URI (Admin only field)</label>
+                    <input-type>onebox</input-type>
+                    <hint>
+                        Please enter the full path of the file, either from the local server or an HTTP location.
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Missing translations

File: dspace/config/submission-forms_cs.xml [hunk 5]
```diff
@@ -3219,15 +3194,15 @@
          Uncomment the example row of the complex input type definition to see this input in the submission UI. -->
     <form-complex-definitions>
         <definition name="contact_person">
-            <input name="givenname" input-type="text" label="Given name"/>
-            <input name="surname" input-type="text" label="Surname"/>
-            <input name="email" input-type="text" label="Email" regex="[^@]+@[^\.@]+\.[^@]+"/>
-            <input name="affiliation" input-type="autocomplete" label="Affiliation"/>
+            <input name="givenname" input-type="text" label="Contact person's given name" required="true"/>
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Missing translations

File: dspace/config/submission-forms_cs.xml [hunk 6]
```diff
@@ -3201,7 +3176,7 @@
         </value-pairs>
         <value-pairs value-pairs-name="hidden_list" dc-term="">
             <pair>
-                <displayed-value>Hidden</displayed-value>
+                <displayed-value>Yes</displayed-value>
```
Comment (milanmajchrak) [inline dspace/config/submission-forms_cs.xml]: Translate

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

**Review by milanmajchrak — DISMISSED** [review]

File: dspace/config/submission-forms.xml [hunk 7]
```diff
@@ -1867,7 +1875,7 @@
                     <input-type>onebox</input-type>
                     <hint>Link to original paper that references this dataset.</hint>
                     <required/>
-                    <regex>http.*</regex>
+                    <regex>The supplied url must start with http/https</regex>
```
Comment (vidiecan) [inline dspace/config/submission-forms.xml]: ?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: dspace/config/submission-forms.xml (hunk 7 above)
Comment (milanmajchrak) [reply dspace/config/submission-forms.xml]: Oh, I didn't realize... Thank you!

**Review by vidiecan — APPROVED** [review]

**Pull request merged**